### DISCUSSION TITLE
Modify macOS build instructions to incorporate universal build functionality for both arm64 + x86_64

### DISCUSSION THREAD

**Discussion by ylluminate** [jamulussoftware/jamulus#3240] [Technical Support & Help]
It appears that the default macOS build instructions only apply to the existing hardware upon which the project is being built. If, say, a universal build is needed this appears to fail. What steps are additionally needed to facilitate a universal binary build for the resultant .app to have both x86_64 + arm64?

**Comment by ann0see** [comment]
I believe that's not verbally documented, but in the .github/autobuild directory there's the code we run for the CI. We do build universal binaries. 

Unfortunately the build produced by Xcode crashes for me. Using make on the cli however works as intended

**Comment by ann0see** [comment]
Hmm. After running xcodebuild, it tries to build opus x86 for arm64:

```
The following build commands failed:
	CompileC .../jamulus/.xcode/Jamulus.build/Release/Jamulus.build/Objects-normal/arm64/x86cpu.o .../jamulus/libs/opus/celt/x86/x86cpu.c normal arm64 c com.apple.compilers.llvm.clang.1_0.compiler (in target 'Jamulus' from project 'Jamulus')
(1 failure)
```
Seems as if we need to change something. 
Which commands were you using?

Edit: xcodebuild ARCHS=x86_64 ONLY_ACTIVE_ARCH=NO build to force x86_64 only works.
